iam/api-client/access.py

```diff
@@ -100,6 +96,31 @@ def set_policy(project_id, policy):
 # [END iam_set_policy]
 
 
+# [START iam_test_permissions]
+def test_permissions(project_id):
+    """Tests IAM permissions of the caller"""
+
+    credentials = service_account.Credentials.from_service_account_file(
+        filename=os.environ['GOOGLE_APPLICATION_CREDENTIALS'],
+        scopes=['https://www.googleapis.com/auth/cloud-platform'])
+    service = googleapiclient.discovery.build(
+        'cloudresourcemanager', 'v1', credentials=credentials)
+
+    permissions = {
+        "permissions": [
+            "resourcemanager.projects.get",
+            "resourcemanager.projects.delete"
+        ]
+    }
+
+    request = service.projects().testIamPermissions(
+        resource=project_id, body=permissions)
+    returnedPermissions = request.execute()
+    print(returnedPermissions)
+    return returnedPermissions
+# [END iam_test_permissions]
```

Comment:
I don't see any assertions in this method. What response is it expecting?

Reply:
Never mind. I saw the name of the method and thought that it was a test method, as opposed to a method that calls the test_permissions API.